src/app/patient-dashboard/common/formentry/patient-transfer.service.ts

```diff
@@ -1,32 +1,43 @@
 import { Injectable } from '@angular/core';
 
-import { BehaviorSubject, Observable } from 'rxjs';
+import { BehaviorSubject } from 'rxjs';
 import * as _ from 'lodash';
-import { FormentryComponent } from './formentry.component';
 import * as moment from 'moment';
+
+import { FormentryComponent } from './formentry.component';
 import { Patient } from '../../../models/patient.model';
-import { FormDataSourceService } from './form-data-source.service';
-import { ConfirmationService } from 'primeng/primeng';
+import { LocalStorageService } from 'src/app/utils/local-storage.service';
+import { DepartmentProgramsConfigService } from 'src/app/etl-api/department-programs-config.service';
+import { SelectDepartmentService } from 'src/app/shared/services/select-department.service';
 
 @Injectable()
 export class PatientTransferService {
   public componentRef: FormentryComponent;
   private patient: Patient;
   private transferState: BehaviorSubject<any> = new BehaviorSubject(null);
 
-  constructor() {
-  }
+  constructor(
+    private localStorageService: LocalStorageService,
+    private selectSetDepartmentService: SelectDepartmentService,
+    private departmentConfigService: DepartmentProgramsConfigService
```

Comment:
Seems like you're not either this dependency or `localStorageService`. Probably better to get rid of them.